src/page/detail/detailCss.js

```diff
@@ -0,0 +1,133 @@
+import {StyleSheet} from 'react-native';
```

Comment:
all files with styles should be called `styles.ts`. 
CSS doesn't exist in React Native